### PR TITLE
My devices page [Approved]

### PR DESCRIPTION
**Pull request by oshhh**
My Devices page added:
1. If no access then gives  "Link Devices" button
2. While fetching devices shows "Loading..."
3. Lists devices, types
4. In case loading devices fails, shows "Reload" button.

[TESTING]:
Written a widget test for happy case. Uses the mock api in SDK for testing.

UI:

### CODE REVIEW

File: home_insurance_app/lib/data/globals.dart [hunk 1]
```diff
@@ -3,13 +3,22 @@ library globals;
...
-  String _clientSecret = _remoteConfig.getString('client_secret');
-  String _enterpriseId = _remoteConfig.getString('enterprise_id');
+Future<SDK> initialiseSDK({test = false}) async {
+  String _clientId;
+  String _clientSecret;
+  String _enterpriseId;
+  if(test) {
+    _clientId = 'clientId';
+    _clientSecret = 'clientSecret';
+    _enterpriseId = 'akashag-step-interns-test';
```
Comment (akashagarwal) [inline home_insurance_app/lib/data/globals.dart:13]: nit: please name this something generic

**Comment by oshhh** [comment]
@googlebot I consent.

**Comment by anjalisoni3655** [comment]
@googlebot I consent.

**Comment by khushi2305** [comment]
@googlebot I consent.

**Pull request merged**